transformers/events/flip_file/cat/transformer.go

```diff
@@ -0,0 +1,59 @@
+package cat
+
+import (
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
+)
+
+type Transformer struct{}
+
+func (Transformer) ToModels(_ string, logs []core.EventLog, db *postgres.DB) ([]event.InsertionModel, error) {
+	var results []event.InsertionModel
+	for _, log := range logs {
+		err := shared.VerifyLog(log.Log, shared.FourTopicsRequired, shared.LogDataRequired)
+		if err != nil {
+			return nil, err
+		}
+
+		what := shared.DecodeHexToText(log.Log.Topics[2].Hex())
+		data := shared.ConvertUint256HexToBigInt(log.Log.Topics[3].Hex())
```

Comment:
mentioned this on the beta PR but this should probably be converted to an address rather than an int

also looks like this PR has several extra commits that hopefully go away after #300 is merged